### PR TITLE
Jython 2.7.0

### PR DESCRIPTION
**Pull request by danslapman**

### CODE REVIEW

File: Library/Formula/jython.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 class Jython < Formula
   homepage "http://www.jython.org"
-  url "https://search.maven.org/remotecontent?filepath=org/python/jython-installer/2.5.3/jython-installer-2.5.3.jar"
-  sha256 "05405966cdfa57abc8e705dd6aab92b8240097ce709fb916c8a0dbcaa491f99e"
+  url "https://search.maven.org/remotecontent?filepath=org/python/jython-installer/2.7.0/jython-installer-2.7.0.jar"
+  sha1 "b08d73b5df7d95e35e7b8dcaf1558025cf24f0f1"
```
Comment (xu-cheng) [inline Library/Formula/jython.rb]: Please use `sha256`.

**Comment by tdsmith** [comment]
Please also remove the `devel` block since it's now stale.

File: Library/Formula/jython.rb (hunk 1 above)
Comment (danslapman) [inline Library/Formula/jython.rb]: search.maven.org does not offer sha256 any more :(

Comment (jacknagel) [inline Library/Formula/jython.rb]: We still need to use it, you just need to confirm the output of `brew fetch jython` yourself.

Comment (danslapman) [inline Library/Formula/jython.rb]: Done.

**Comment by tdsmith** [comment]
The `inreplace` in the `install` method is failing; it looks like it can be deleted?

**Comment by danslapman** [comment]
Okay, done...

**Comment by tdsmith** [comment]
Thanks for the PR!

**Pull request closed without merging**